docker-compose.yml

```diff
@@ -488,17 +488,36 @@ services:
     shm_size: *shm-size
     ulimits: *ulimits
     environment: &cuda-environment
```

Comment:
can we call this `cuda-minimal-environment` or `cuda-cpp-environment`, I was confused when we did not use it on `ubuntu-cuda-python` but for python we have to enable a bunch of other things, the current name is slightly confusing.

Reply:
I was planning to use it on both jobs but as you said that is not possible, so I will remove it because it is only used in this job.